primo2/networks.py

```diff
@@ -69,6 +69,29 @@ def get_node(self, node_name):
             return self.node_lookup[node_name]
         except KeyError:
             raise Exception("There is no node with name {} in the BayesianNetwork".format(node_name))
+            
+    def change_node_values(self, node, newValues):
```

Comment:
Not a big thing, but PEP8 recommends the use of variables names with underscores, like so: `new_value`. The same applies for the methods below.

Reply:
You are right, I am still stuck with camelCase for those. We are not really consistent in our codebase right now, but it would be good to just pick one style and stick with it. I am fine with either, but maybe we should create an issue+PR making the entire codebase consistent.

Reply:
Ok, that is a good idea.